frontend/src/pages/NewRunSwitcher.tsx

```diff
@@ -16,7 +16,8 @@ function NewRunSwitcher(props: PageProps) {
   const namespace = React.useContext(NamespaceContext);
 
   const urlParser = new URLParser(props);
-  const originalRunId = urlParser.get(QUERY_PARAMS.cloneFromRun);
+  const originalRunId =
+    urlParser.get(QUERY_PARAMS.fromRunId) || urlParser.get(QUERY_PARAMS.cloneFromRun);
```

Comment:
This is a bit confusing: Why we expect `fromRunId` in new run page? Shouldn't `cloneFromRun` be sufficient?

Reply:
If we click `Clone run` at the run details page, it will direct to the clone run page it will direct to the clone run page.

If we click `Create run` at the pipeline details page (without pipeline version), it only has run ID, and the action behind create run is using `fromRunId` under `QUERY_PARAM`. If we only use cloneFromRun here. It will fail to get `RunID`.

Reply:
> If we click Create run at the pipeline details page (without pipeline version), it only has run ID, and the action behind create run is using fromRunId under QUERY_PARAM.

This statement is partially accurate. See the comment in https://github.com/kubeflow/pipelines/blob/master/frontend/src/pages/NewRun.tsx#L650-L655 for the use case of `fromRunId`. It will be great to reuse same comment here to reduce confusion in the future.